.github/workflows/changelog_reminder.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  pull_request:
+    types: [opened, ready_for_review]
```

Comment:
How about also at `review_requested`?

Reply:
I assumed ready_for_review and review_requested would happen at about the same time, and I don't want it being repetitiveve